client/app/pod/queue.controller.js

```diff
@@ -0,0 +1,24 @@
+angular.module('throwcast.queue')
+
+.controller('QueueController', function ($scope, $http) {
+  $scope.h1 = 'Podcast Queue';
+  $scope.podcastQueue;
+  $scope.link =  $scope.podcastQueue[0].link || '';
```

Comment:
This will always be undefined when the controller loads. You can just remove this.